### PR TITLE
Feature/active today

### PR DESCRIPTION
**Pull request by johantjellden**

### CODE REVIEW

File: src/main/java/com/hedvig/notificationService/customerio/events/ActivationDateTodayEvent.kt [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package com.hedvig.notificationService.customerio.events
+
+import com.fasterxml.jackson.annotation.JsonTypeInfo
+import com.fasterxml.jackson.databind.PropertyNamingStrategy
+import com.fasterxml.jackson.databind.annotation.JsonNaming
+
+@JsonNaming(PropertyNamingStrategy.SnakeCaseStrategy::class)
+@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, include = JsonTypeInfo.As.PROPERTY, property = "name", visible = true)
+data class ActivationDateTodayEvent(
```
Comment (meletios) [inline src/main/java/com/hedvig/notificationService/customerio/events/ActivationDateTodayEvent.kt:9]: Is the naming a bit confusing? Shall we following the naming that we are using in the events for far. 
```suggestion
data class ContractsActivatedTodayEvent(
```

File: src/main/java/com/hedvig/notificationService/customerio/events/ActivationDateTodayEvent.kt [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+    val activeToday: List<Contract>,
+    val activeInFuture: List<Contract>
```
Comment (meletios) [inline src/main/java/com/hedvig/notificationService/customerio/events/ActivationDateTodayEvent.kt:11]: Shall this list be on a different event?

Comment (johantjellden) [reply src/main/java/com/hedvig/notificationService/customerio/events/ActivationDateTodayEvent.kt:11]: In the event we want to have both the contracts that are active from today, and contracts that will become active in the future. 
These events are sent to customer.io and we want to include enough information in them to be able to render messages.

File: src/main/java/com/hedvig/notificationService/customerio/events/ActivationDateTodayEvent.kt (hunk 1 above)
Comment (johantjellden) [reply src/main/java/com/hedvig/notificationService/customerio/events/ActivationDateTodayEvent.kt:9]: Good idea.

File: src/main/java/com/hedvig/notificationService/customerio/ContractInfo.kt [hunk 3]
```diff
@@ -9,8 +9,18 @@ data class ContractInfo(
...
-    SwedishHouse,
-    SwedishApartment
+    NorwegianHomeContent {
+        override val typeName: String = "innbo";
+    },
+    NorwegianTravel {
+        override val typeName: String = "reise"
+    },
+    SwedishHouse {
+        override val typeName: String = "swedish_house"
```
[the remark this answers is not in the document]
Comment (johantjellden) [reply src/main/java/com/hedvig/notificationService/customerio/ContractInfo.kt:19]: No it is not to late yet.

**Pull request merged**